src/main/java/com/icthh/xm/ms/dashboard/service/dto/BulkDashboardResult.java

```diff
@@ -0,0 +1,36 @@
+package com.icthh.xm.ms.dashboard.service.dto;
+
+import lombok.experimental.UtilityClass;
+
+import static com.icthh.xm.ms.dashboard.service.dto.BulkOperationStatus.CREATED;
+import static com.icthh.xm.ms.dashboard.service.dto.BulkOperationStatus.DELETED;
+import static com.icthh.xm.ms.dashboard.service.dto.BulkOperationStatus.FAILED;
+import static com.icthh.xm.ms.dashboard.service.dto.BulkOperationStatus.UPDATED;
+
+@UtilityClass
+public class BulkDashboardResult {
```

Comment:
Instead of this class please consider next option: 
Add function to enum BulkOperationStatus:

```
public enum BulkOperationStatus {
    CREATED, UPDATED, DELETED, FAILED;

    public final BulkDashboardItemStatus toItemStatus(DashboardDto dashboardDto) {
        return BulkDashboardItemStatus.builder()
                                      .dashboardItem(dashboardDto)
                                      .status(this).build();
    }
}
```

and then use as:

`itemStatuses.add(CREATED.toItemStatus(dashboardDto));`

Reply:
Good idea, I will consider this optimization